### PR TITLE
fix(api-client): set param example value

### PR DESCRIPTION
**Pull request by antlio**
this pr sets back param example values, which displays the first occurence:

image

### CODE REVIEW

File: packages/api-client/src/store/workspace.ts [hunk 1]
```diff
@@ -178,7 +178,11 @@ const createParamInstance = (param: OpenAPIV3_1.ParameterObject) =>
   createRequestExampleParameter({
     key: param.name,
     value:
-      param.schema && 'default' in param.schema ? param.schema.default : '',
+      param.schema && 'default' in param.schema
+        ? param.schema.default
+        : param.schema && 'examples' in param.schema
+          ? param.schema.examples[0]
```
Comment (hanspagel) [inline packages/api-client/src/store/workspace.ts:184]: In theory, examples could be an empty array. 🤡

**Review by hanspagel — CHANGES_REQUESTED** [review]

File: packages/api-client/src/store/workspace.ts (hunk 1 above)
Comment (antlio) [reply packages/api-client/src/store/workspace.ts:184]: it'll just display no value right now, and `__PLANETID__` in the reference example. wdyt?

Comment (marclave) [reply packages/api-client/src/store/workspace.ts:186]: we can probably down the road have a helper function to get default values from openapi spec, i feel like theres a ton of theoretical limits we hit ;)

**Review by marclave — APPROVED** [review]
i dreamed of this last night and woke up and its here!!

**Pull request merged**